### DISCUSSION TITLE
Executing script without opening REPL

### DISCUSSION THREAD

**Discussion by awa5114** [waf/CSharpRepl#303] [Q&A] [answered]
Is there a way to simply execute a script using `csharprepl.exe` without opening the REPL afterwards?

**Comment by awa5114** [comment, marked as answer]
I found a potential workaround namely adding the following at the end of the script:

```
Environment.Exit(0);
```

Would be nice to have a built-in way to just run the scripts at the CLI level without polluting the source code,

**Comment by WeihanLi** [comment]
why not use `dotnet-script` for executing script?
